File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.MessagingService.Models/save-notification_param.ts


import { ISaveNotification_Param } from './isave-notification_param';

export class SaveNotification_Param implements ISaveNotification_Param {
  
	public ContextTypeID: number = 0;
	public ContextID: number = 0;
	public NotificationMessageID?: number;
	public NotificationMessage: string = "";
	public NotificationSubject: string = "";
  
}

File: GTTS/INL.MessagingService/Models/SaveNotification_Param.cs

namespace INL.MessagingService.Models
{
    public class SaveNotification_Param : ISaveNotification_Param
    {
        public int ContextTypeID { get; set; }
        public long ContextID { get; set; }
        public int? NotificationMessageID { get; set; }
        public string NotificationMessage { get; set; }
        public string NotificationSubject { get; set; }
    }
}
